object_analytics_rviz/src/marker_publisher.cpp

```diff
@@ -47,25 +51,29 @@ class MarkerPublisher : public rclcpp::Node
   MarkerPublisher()
   : Node("marker_publisher")
   {
-    loc_subscription_ = this->create_subscription<LocalizationMsg>(
-      "/object_analytics/localization", std::bind(&MarkerPublisher::loc_callback, this, _1));
-    tra_subscription_ = this->create_subscription<TrackingMsg>(
-      "/object_analytics/tracking", std::bind(&MarkerPublisher::tra_callback, this, _1));
-    marker_subscription_ = this->create_subscription<LocalizationMsg>(
-      "/object_analytics/localization", std::bind(&MarkerPublisher::marker_callback, this, _1));
+    loc_performance_subscription_ = this->create_subscription<LocalizationMsg>(
+      "/object_analytics/localization",
```

Comment:
better to replace "/object_analytics/localization" with Const::kTopicLocalization